frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-details-participant-list-table/common-details-participant-list-table.component.scss

```diff
@@ -76,6 +76,46 @@
     .enable-column {
       width: 20%;
       text-align: center;
+
+      ::ng-deep .mat-mdc-slide-toggle {
```

Comment:
The `::ng-deep .mat-mdc-slide-toggle` styles look like duplicate code, and it's pretty likely to get used elsewhere as we go... can you try moving this to `styles.scss` or something like that to generalize it?